lib/chef/deprecated.rb

```diff
@@ -156,6 +156,16 @@ def target
       end
     end
 
+    class PropertyNameCollision < Base
+      def id
+        10
```

Comment:
I can bump mine to 11, didn't know there was something already in flight. Didn't know if we wanted to start a registry of these somewhere that we grab from before we implement.